### DISCUSSION TITLE
Is there a roadmap somewhere?

### DISCUSSION THREAD

**Discussion by flowt-au** [FalkorDB/FalkorDB#590] [General]
Hi. I am wondering if there is a roadmap for FalkorDb?
The feature I am most interested in is nested json in nodes, which topped the survey in terms of "missing features". I am wondering what the plans are in this regard?
Thanks for your great work.
Murray

**Comment by swilly22** [comment]
Hi @flowt-au please see: https://github.com/FalkorDB/FalkorDB/pull/658

**Reply by flowt-au** [reply]
WooHoo!! I will be able to greatly simplify my code once this is available. Thank you!

**Comment by xtrasmal** [comment]
So is there or isn't there a roadmap?
